### PR TITLE
[ALLUXIO-1553] remove redundant documentation with @return

### PR DESCRIPTION
**Pull request by vincentpham1991**
https://alluxio.atlassian.net/browse/ALLUXIO-1553

### CODE REVIEW

**Comment by gpang** [comment]
LGTM

**Comment by ifcharming** [comment]
ok to test

File: core/server/src/main/java/alluxio/master/MasterContext.java [hunk 1]
```diff
@@ -34,19 +34,13 @@ private MasterContext() {} // to prevent initialization
   private static MasterSource sMasterSource = new MasterSource();
 
   /**
-   * Returns the one and only static {@link Configuration} object which is shared among all classes
-   * within the master process.
-   *
```
Comment (calvinjia) [inline core/server/src/main/java/alluxio/master/MasterContext.java]: Do you mind adding the info in the original statement into the `@return`?

**Comment by gpang** [comment]
Jenkins, add to whitelist

**Pull request merged**